apps/api/src/domain/models/book/entities/errors/PublishedBookHasNoPagesError.ts

```diff
@@ -0,0 +1,11 @@
+import { InternalError } from '../../../../../lib/errors/InternalError';
+import formatAggregateType from '../../../../../view-models/presentation/formatAggregateType';
+import { AggregateType } from '../../../../types/AggregateType';
+
+export default class PublishedBookHasNoPagesError extends InternalError {
+    constructor() {
+        super(
+            `A ${formatAggregateType(AggregateType.book)} cannot be published and yet have no pages`
```

Comment:
or "cannot be published with zero pages" or "cannot be published without pages"

Reply:
thanks. I went with the latter.